05-Jason-Luo/wk02/2-tue/Paul_Dirac/trivia.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Paul Dirac</title>
+    <link rel="stylesheet" href="css/display.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <ul>
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="./bio.html">Bio</a></li>
+                <li><a href="./trivia.html">Trivia</a></li>
+                <li><a href="./hometown.html">Hometown</a></li>
+            </ul>
+            <h1>Trivia about Paul Dirac</h1>
+        </header>
+
+        <main>
+            <div class="trivia">
+                <div class="trivia-item">
+                    <div class="picture">
+                        <img src="images/paul1.jpeg" alt="">
+                    </div>
+                    <div class="triviahover">
+                        <p> wwerwer</p>
+                    </div>
+
+                </div>
+                <div class="trivia-item">
+                    <div class="picture">
+                        <img src="images/paul1.jpeg" alt="">
+                    </div>
+                    <div class="triviahover">
+                        <p>He is credited with the discovery of relativistic equation for the electron and for drafting
+                            the most general theory of quantum mechanics; which are also named after him.</p>
```

Comment:
```suggestion
                        <p>
                            He is credited with the discovery of relativistic equation for the electron and for drafting
                            the most general theory of quantum mechanics; which are also named after him.
                        </p>
```
This is how i prefer to indent my tags when they have multiple lines of content.